src/Jackett.Common/Indexers/Abstract/AvistazTracker.cs

```diff
@@ -99,37 +101,36 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
 
             try
             {
-                CQ dom = response.Content;
-                var rows = dom["table:has(thead) > tbody > tr"];
+                var parser = new HtmlParser();
+                var dom = parser.ParseDocument(response.Content);
+                var rows = dom.QuerySelectorAll("table:has(thead) > tbody > tr");
                 foreach (var row in rows)
                 {
-                    var qRow = row.Cq();
+                    
                     var release = new ReleaseInfo
                     {
                         MinimumRatio = 1,
                         MinimumSeedTime = 172800 // 48 hours
                     };
 
-                    var qLink = qRow.Find("a.torrent-filename");
-                    ;
+                    var qLink = row.QuerySelector("a.torrent-filename");
                     release.Title = qLink.Text().Trim();
-                    release.Comments = new Uri(qLink.Attr("href"));
+                    release.Comments = new Uri(qLink.GetAttribute("href"));
                     release.Guid = release.Comments;
 
-                    var qDownload = qRow.Find("a.torrent-download-icon");
-                    ;
-                    release.Link = new Uri(qDownload.Attr("href"));
+                    var qDownload = row.QuerySelector("a.torrent-download-icon");
+                    release.Link = new Uri(qDownload.GetAttribute("href"));
 
-                    var dateStr = qRow.Find("td:eq(3) > span").Text().Trim();
+                    var dateStr = row.QuerySelector("td:eq(3) > span").Text().Trim();
```

Comment:
`:eq` is not working. => https://github.com/AngleSharp/AngleSharp/issues/183
You can use `nth-of-type(4)` instead. Notice the index + 1
Apply to all occurrences in other PRs too.